src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_course?serverTimezone=Europe/Warsaw&useSSL=False&allowPublicKeyRetrieval=true
+spring.datasource.url=jdbc:mysql://localhost:3306/jdp?serverTimezone=Europe/Warsaw&useSSL=False&allowPublicKeyRetrieval=true
```

Comment:
Celowo utworzyłeś nową bazę danych?

Reply:
Przy testowaniu utworzyłem sobie nową bazę danych i przed commitem miałem dać kodilla_course i zapomniałem. Już to poprawiłem